.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Generated from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cudf/labels
+
+cuDF (Python):
+  - 'python/**'
+  - 'notebooks/**'
+  
+libcudf:
+  - 'cpp/**'
+
+CMake:
+  - '**/CMakeLists.txt'
+  - '**/cmake/**'
+  
+cuDF (Java):
+  - 'java/**'
+  
+Ops:
+  - '.github/**'
+  - 'ci/**'
+  - 'conda/**'
```

Comment:
We have a `conda` label it would be nice to put this under.

Reply:
Agreed this should be it's own labeled section